images/spire/config/main.tf

```diff
@@ -22,7 +27,8 @@ output "config" {
     }
     accounts = module.accts.block
     entrypoint = {
-      command = "/usr/bin/spire-${var.name}"
+      ## agent and server require `run` while oidc-discovery-provider doesn't, this condition tries to handle that
+      command = var.name == "agent" || var.name == "server" ? "/usr/bin/spire-${var.name} run" : "/usr/bin/${var.name}"
```

Comment:
Following the same idea from the previous comment:
```terraform
  command = var.command
```

Reply:
Perfect, will do, thanks so much!